index.src.html

```diff
@@ -705,73 +759,75 @@ <h4 id="fetching">Fetching</h4>
+          null.
 
-          1.  If |connection|'s [=connection/IP address space=] is
-              [=IP address space/less public=] than |clientAddressSpace|, then
-              return a [=network error=].
+      1.  If |request|'s [=request/policy container=] is null, then return null.
 
-          1.  Return null.
+          NOTE: If |request|'s [=request/policy container=] is null, then PNA
+          checks do not apply to |request|. Users of the [=fetch=] algorithm
+          should take care to either set |request|'s [=request/client=] to an
+          [=environment settings object=] with a non-null [=environment settings
+          object/policy container=] and let [=fetch=] initialize |request|'s
+          [=request/policy container=] accordingly, or to directly set
+          |request|'s [=request/policy container=] to a non-null value.
 
-          NOTE: If |request|'s [=request/policy container=] is null, then PNA checks
-          do not apply to |request|. Users of the [=fetch=] algorithm should take
-          care to either set |request|'s [=request/client=] to an
-          [=environment settings object=] with a non-null
-          [=environment settings object/policy container=] and let [=fetch=]
-          initialize |request|'s [=request/policy container=] accordingly, or to
-          directly set |request|'s [=request/policy container=] to a non-null value.
+      1.  Let |clientAddressSpace| be |request|'s [=request/policy container=]'s
+          [=policy container/IP address space=]
 
-      1.  Let |clientAddressSpace| be |request|'s
-          [=request/policy container=]'s
-          [=policy container/IP address space=].
+      1.  If |request|'s [=request/client=] is not a [=secure context=]
+          (including if it is null), and |connection|'s [=connection/IP address
+          space=] is [=IP address space/less public=] than |clientAddressSpace|,
+          then return a [=network error=].
 
       1.  If |request|'s [=request/target IP address space=] is not null, then:
 
           1.  If |connection|'s [=connection/IP address space=] is not equal to
               then |request|'s [=request/target IP address space=], then return
               a [=network error=].
 
-          1.  If |response|'s [=response/IP address space=] is not
-              [=IP address space/less public=] than |clientAddressSpace|, then
-              return a [=network error=].
-
           1.  Return null.
 
-      1.  If |response|'s [=response/IP address space=] is
+      1.  If |connection|'s [=connection/IP address space=] is
           [=IP address space/less public=] than |clientAddressSpace|, then:
 
               1.  Let |error| be a [=network error=].
 
-              1.  Set |error|'s [=response/IP address space=] property to
-                  |response|'s [=response/IP address space=].
+              1.  Set |error|'s [=connection/IP address space=] property to
```

Comment:
`|error|` is a response:

```suggestion
              1.  Set |error|'s [=response/IP address space=] property to
```